app/src/main/java/org/oppia/android/app/help/HelpItemViewModel.kt

```diff
@@ -13,6 +18,7 @@ class HelpItemViewModel(
     if (title == activity.getString(R.string.frequently_asked_questions_FAQ)) {
       val routeToFAQListener = activity as RouteToFAQListListener
       routeToFAQListener.onRouteToFAQList()
+      activity.help_item_text_view.isEnabled = false
```

Comment:
/cc @rt4914 

I have 3 main concerns with this approach:
1. The mechanism for reenabling the button is in an entire different file, and assumes a particular lifecycle order that may not be guaranteed. One example where this may break (in theory): if the button hangs due to a long operation. It will stay disabled and won't be reenabled since onResume won't be called (unless the user navigates away, backgrounds & foregrounds the app, or triggers a configuration change e.g. via a rotation). By being in a different file, it's likely people will forget to add that part which may result in broken & potentially difficult-to-debug scenarios arising.
2. We aren't tying the "enable/re-enable" state to the desired outcome. We have to assume that operations can fail if we assume they might take too long. For this reason, we may actually need to consider introducing loading interstitials that are tied directly to DataProviders/LiveDatas to help indicate when it's time to complete the transition. That approach may help us generalize the pattern, and make it more robust by introducing timeouts to ensure the button does eventually get re-enabled. This also might help guard against a potential broken scenario where the user clicks a button and, while rotating, tries clicking it again (since the current code allows this to happen).
2. As written, it's not generalizable. Ideally, we'd come up with a plug-and-play solution that can be easily adapted to multiple button scenarios with minimal effort or as the default behavior to try and cover as many scenarios where this can happen as possible.